R/deps.R

```diff
@@ -17,7 +17,10 @@
 #'   and is the default. `FALSE` is shorthand for no dependencies (i.e.
 #'   just check this package, not its dependencies).
 #' @param quiet If `TRUE`, suppress output.
-#' @param upgrade If `TRUE`, also upgrade any of out date dependencies.
+#' @param upgrade If `NA` and interactive prompt to upgrade out of date
```

Comment:
This is a bit hard to read I think, bc interactive could an adjective, as  in "interactive prompt". I had to read  it three times  to understand.